components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__phenotype.xml

```diff
@@ -578,10 +578,13 @@
 }
 #current-phenotype-selection .summary-item label {
   width: auto;
-  margin-right: 1em;
+  margin-right: 0.3em;
+}
+#current-phenotype-selection .summary-item .tools .xHelpButton {
+  margin : 0.6em;
 }
 #current-phenotype-selection .summary-item .tools {
-  color: $theme.textPrimaryColor;
+  color: $theme.textColor;
```

Comment:
If we remove the `hint` css class for the `tools` element as sugested in the previous comment, the color will default to `$theme.textColor`, so this line doesn't need to be added (just remove `$theme.textPrimaryColor`).